.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": "airbnb-base",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+    "rules": {
+        "semi": ["error", "always"],
+        "quotes": ["error", "double"],
+        "linebreak-style" : ["error", "windows"]
+    }
+};
```

Comment:
new line

Reply:
Add new line at the end of the file in the commit below:
https://github.com/BlackTruth/virtual-keyboard/commit/bace22c72e918a327db89ea7670c0dcf87c2bd6f?diff=unified#diff-e4403a877d80de653400d88d85e4801a